providers/os/connection/device/device_manager.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package device
+
+import "go.mondoo.com/cnquery/v11/providers/os/connection/device/shared"
+
+type DeviceManager interface {
```

Comment:
we can change this interface in the future to make this work better with mac/windows if needed